app/cafe.py

```diff
@@ -0,0 +1,23 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError
+        else:
+            if visitor["vaccine"]["expiration_date"] < date.today():
+                raise OutdatedVaccineError
+
+            else:
+                if not visitor["wearing_a_mask"]:
```

Comment:
It would be great to check if `wearing_a_mask `exists in visitor to prevent KeyError